src/app/server/Server.cpp

```diff
@@ -162,14 +160,14 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
 
     err = mListener.Init(&mTransports);
     SuccessOrExit(err);
-    mGroupsProvider.SetListener(&mListener);
+    mGroupsProvider->SetListener(&mListener);
```

Comment:
I think we ought to have a null check checks for this and the other injected members somewhere.  One null check in init is probably fine.

Reply:
See at the top of the function:

```
    VerifyOrExit(initParams.persistentStorageDelegate != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
    VerifyOrExit(initParams.groupDataProvider != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
    VerifyOrExit(initParams.accessDelegate != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);

```

Reply:
Oh, apologies.  There it is.

Thank you.